docs/indepth/installation.rst

```diff
@@ -359,19 +361,43 @@ The following commands will install the two repositories and the ``flocker-node`
 Paste them into a root console on the target node:
 
 .. task:: install_flocker centos-7
-   :prompt: [root@node]#
-
-
-.. _centos-7-install:
-
-Post installation configuration for Fedora 20 and CentOS 7
-----------------------------------------------------------
+   :prompt: [root@centos]#
 
 Installing ``flocker-node`` will automatically install Docker, but the ``docker`` service may not have been enabled or started.
 To enable and start Docker, run the following commands in a root console:
 
 .. task:: enable_docker
-   :prompt: [root@node]#
+   :prompt: [root@centos]#
+
+
+.. _ubuntu-14.04-install:
+
+Installing on Ubuntu 14.04
+--------------------------
+
+.. note:: The following commands all need to be run as root on the machine where ``clusterhq-flocker-node`` will be running.
+
+.. To test this:
```

Comment:
:question: AFAIK you have been testing this with a different setup. If you think it'd be useful document that (in note form?).